From 0b47fca08942c8728f5a8c26ebed72052ee48722 Mon Sep 17 00:00:00 2001
From: Peter O'Connor <peter@solus-project.com>
Date: Fri, 3 Mar 2017 16:18:52 +1100
Subject: [PATCH 2/2] data: Provide a local repo configuration

This allows for the easy (and standardised) set up for using local
packages with solbuild.
---
 data/local-unstable-x86_64.profile | 33 +++++++++++++++++++++++++++++++++
 1 file changed, 33 insertions(+)
 create mode 100644 data/local-unstable-x86_64.profile

diff --git a/data/local-unstable-x86_64.profile b/data/local-unstable-x86_64.profile
new file mode 100644
index 0000000..59e0641
--- /dev/null
+++ b/data/local-unstable-x86_64.profile
@@ -0,0 +1,33 @@
+#
+# local-unstable-x86_64 configuration
+#
+# Build Solus packages using the unstable repository image.
+# This is the default profile for the Solus build server and developers.
+#
+# Do not make changes to this file. solbuild is implemented in a stateless
+# fashion, and will load files in a layered mechanism. If you wish to edit
+# this profile, copy to /etc/solbuild/.
+#
+# It is generally advisable to create a *new* profile name in /etc, because
+# we will load /etc/ before /usr/share. Thus, profiles with the same name
+# in /etc/ are loaded *first* and will override this profile.
+#
+# Of course, if that's what you intended to do, then by all means, do so.
+
+image = "unstable-x86_64"
+
+# If you have a local repo providing packages that exist in the main
+# repository already, you should remove the repo, and re-add it *after*
+# your local repository:
+remove_repos = ['Solus']
+add_repos = ['Local','Solus']
+
+# A local repo with automatic indexing
+[repo.Local]
+uri = "/var/lib/solbuild/local"
+local = true
+autoindex = true
+
+# Re-add the Solus unstable repo
+[repo.Solus]
+uri = "https://packages.solus-project.com/unstable/eopkg-index.xml.xz"
-- 
2.12.0

